### PR TITLE
Add tvOS support in Xcode and podspec

### PR DESCRIPTION
**Pull request by asmallteapot**
- Add "GRDBtvOS" target and scheme in the Xcode project
- Add tvOS as a supported platform in podspec
- Add tvOS to a platform compatibility check

### CODE REVIEW

**Comment by groue** [comment]
Hello @asmallteapot, thanks for this PR! I'll be off a few days, and I'll look at it when I'm back.

File: GRDB/Core/Database.swift [hunk 1]
```diff
@@ -307,7 +307,7 @@ extension Database {
                 db.configuration.trace!(sql)
             }, dbPointer)
         #else
-            if #available(iOS 10.0, OSX 10.12, watchOS 3.0, *) {
+            if #available(iOS 10.0, OSX 10.12, watchOS 3.0, tvOS 10.0, *) {
```
Comment (groue) [inline GRDB/Core/Database.swift]: This is important, thanks. Now, did you check if other availability checks in GRBD should be updated as well?

File: GRDB.swift.podspec [hunk 2]
```diff
@@ -11,6 +11,7 @@ Pod::Spec.new do |s|
 	
 	s.ios.deployment_target = '8.0'
 	s.osx.deployment_target = '10.9'
+	s.tvos.deployment_target = '9.0'
```
Comment (groue) [inline GRDB.swift.podspec]: How did you pick that version?

**Comment by groue** [comment]
Any news, @asmallteapot?

**Comment by asmallteapot** [comment]
Hi @groue, thanks for your feedback! I’ll address your questions now and hope to make the code changes you’ve requested later today.

File: GRDB.swift.podspec (hunk 2 above)
Comment (asmallteapot) [reply GRDB.swift.podspec]: `9.0` is the first public version of the tvOS platform – https://en.wikipedia.org/wiki/tvOS#History

File: GRDB/Core/Database.swift (hunk 1 above)
Comment (asmallteapot) [reply GRDB/Core/Database.swift]: This was the only change needed to get my demo code working. I’ll check the other availability macros and make appropriate changes.

**Comment by groue** [comment]
Great news! Version 2.4.1 has shipped since, so you may have to merge or rebase your pull request. I'm looking forward to hearing from you and tvOS.

**Comment by groue** [comment]
Closing due to inactivity.

**Pull request closed without merging**